### PR TITLE
Fix various typos in code

### PR DESCRIPTION
**Pull request by marcwrobel**

### CODE REVIEW

File: extensions/amazon-lambda/event-server/src/main/java/io/quarkus/amazon/lambda/runtime/MockEventServer.java [hunk 1]
```diff
@@ -104,10 +104,10 @@ public void setupRoutes() {
         router.route(INVOCATION + ":requestId" + AmazonLambdaApi.API_PATH_REQUEUE).handler(this::handleRequeue);
         router.route(INVOCATION + ":requestId" + AmazonLambdaApi.API_PATH_RESPONSE).handler(this::handleResponse);
         router.route(INVOCATION + ":requestId" + AmazonLambdaApi.API_PATH_ERROR).handler(this::handleError);
-        defaultHanderSetup();
+        defaultHandlerSetup();
     }
 
-    protected void defaultHanderSetup() {
+    protected void defaultHandlerSetup() {
```
Comment (gsmet) [inline extensions/amazon-lambda/event-server/src/main/java/io/quarkus/amazon/lambda/runtime/MockEventServer.java:110]: Note for myself: we will need a note in the migration guide for this one.

File: extensions/panache/hibernate-reactive-rest-data-panache/deployment/pom.xml [hunk 2]
```diff
@@ -10,7 +10,7 @@
     </parent>
 
     <artifactId>quarkus-hibernate-reactive-rest-data-panache-deployment</artifactId>
-    <name>Quarkus - Hibernate Reactive Reactive data with Panache - Deployment</name>
+    <name>Quarkus - Hibernate Reactive data with Panache - Deployment</name>
```
Comment (gsmet) [inline extensions/panache/hibernate-reactive-rest-data-panache/deployment/pom.xml:13]:
```suggestion
    <name>Quarkus - Hibernate Reactive REST Data with Panache - Deployment</name>
```
And probably worth checking the parent and runtime modules are consistent with this naming.

**Review by gsmet — CHANGES_REQUESTED** [review]
Thanks! I just spotted something that needs fixing in a POM (and maybe several of them to make things consistent).

File: extensions/panache/hibernate-reactive-rest-data-panache/deployment/pom.xml (hunk 2 above)
Comment (marcwrobel) [reply extensions/panache/hibernate-reactive-rest-data-panache/deployment/pom.xml:13]: Just checked the parent / runtime module and they are properly named. I also check the pattern `reactive[ -]data[^sb]` (`sb` to exclude datasource and database) and did not find anything except `REST Reactive data` (fixed).

**Review by gsmet — APPROVED** [review]

**Review by mamadaliev — APPROVED** [review]

**Pull request merged**